src/engine/QueryExecutionTree.cpp

```diff
@@ -268,6 +285,7 @@ nlohmann::json QueryExecutionTree::writeResultAsSparqlJson(
   };
 
   for (size_t rowIndex = offset; rowIndex < upperBound; ++rowIndex) {
+    // Due to be an 'nlohmann' object, object keys are alphabetically sorted!
```

Comment:
Replace by `//TODO: `ordered_json` entries are ordered alphabetically, but insertion order would be preferable.